clash-vexriscv-sim/app/VexRiscvChainSimulation.hs

```diff
@@ -134,19 +182,22 @@ main = do
           <> header "vex-riscv-chain-simulation - a test for JTAG chaining"
       )
 
+type PrintClockCycles = Bool
+
 runSampling ::
+  PrintClockCycles ->
   DebugConfiguration ->
   (Handle, Handle) ->
   Signal System (CpuSignals, CpuSignals) ->
   IO ()
```

Comment:
I don't think this is better than a simple haddock comment for a case with a simple boolean. type synonyms also introduce mental overhead

Reply:
Done